vue/sbc-common-components/src/util/common-util.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Place to put all the custom utility methods
+ */
+export function getBoolean (value: boolean | string | number): boolean {
```

Comment:
most of the cases are handled by if condition itself i think... https://basarat.gitbooks.io/typescript/docs/javascript/truthy.html does it fit ur case?